target/linux/ath79/base-files/etc/board.d/02_network

```diff
@@ -148,6 +148,11 @@ ath79_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"0@eth1" "1:lan:4" "2:lan:3" "3:lan:2" "4:lan:1"
 		;;
+	tplink,tl-wr842-v2)
+		ucidef_set_interfaces_lan_wan "eth1.1" "eth0"
```

Comment:
Hi, I think
ucidef_set_interface_wan "eth0"
is enough, since lan is set by the switch.

Reply:
I will give it a go since I reused ar71xx configuration for network.

Reply:
It works.
Will push it today